data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelProtoCodec.java

```diff
@@ -306,38 +270,6 @@ private Map<String, List<ScopeSpans>> splitScopeSpansByTraceId(final List<ScopeS
             return result;
         }
 
-        private List<Span> parseInstrumentationLibrarySpans(final List<InstrumentationLibrarySpans> instrumentationLibrarySpansList,
```

Comment:
Same question: Why is this removed entirely?

Reply:
This API removed because the underlying function (` instrumentationLibrarySpans.getInstrumentationLibrary()`) is not valid anymore